README.md

```diff
@@ -19,3 +21,27 @@ $ pre-commit run --all-files  # run on everything
 $ pre-commit run              # run on staged
 $ git commit -am "test"       # run pre-commit and - if successful - commit
 ```
+
+### Issues and client integration
+
+If pre-commit is not an available command, you may need to look into platform-specific configuration. Generally, you'll need to be in a shell environment that has access to python and with pre-commit installed as directed in the above sections. Typical issues include a misconfigured PATH variable and not having python available. For specific help on integrating with various clients and on various operation systems, see the sections below. If you solve other client integration problems for your favorite workflow, please expand this section in a pull request.
+
+#### Shell Integration on Windows
+
+- Make sure git is set up for normal shell use (a git installation option) if you want to use cmd or powershell
+- I have had success using the Anaconda Powershell Prompt and a conda environment with pre-commit installed
+
+#### Git Bash Integration on Windows
+
+- Add the following to your `~/.bash_profile`: `alias python='winpty python'`, to allow python to run without hangining.
+- If using conda, run `$ conda init bash`, using the conda.exe in your `~/miniconda3/scripts` folder. You may also want to set this up to `conda activate` your `pre-commit` environment.
+- restart your shell after doing the above
+
+#### TwinCAT Integrated Git on Windows
+
+- Someone needs to investigate this one. I don't have this running locally yet and am happy enough with the shells for now.
```

Comment:
Who would be a good candidate here? I think a lot of our team is using this integration.

Reply:
I'm not sure who is using this aside from Alex